src/prompts/styles/_anims.scss

```diff
@@ -74,7 +74,7 @@
     }
 
     100% {
-        transform: translateY(0, 0, 0);
```

Comment:
here same as above, but it correctly gives 1st arg 0, so, these animations did work